.serena/project.yml

```
@@ -1,13 +1,18 @@
+
+
```

YAMLlint (1.38.0)

[error] too many blank lines (2 > 0)

(empty-lines)